template_scripts/packages_fc.list

```diff
@@ -72,6 +74,7 @@ zip
 --exclude=mlocate
 --exclude=nss-mdns
 --exclude=openssh-server
+--exclude=pipewire
```

Comment:
We are thinking to use `pipewire` to replace our Pulseaudio implementation so I guess it's too early for this one.

Reply:
Thanks for review, fixed